department-of-veterans-affairs/caseflow
# Enable Substitution for Hearing Appeals w/ Feature Toggle

Connects [CASEFLOW-1485](https://vajira.max.gov/browse/CASEFLOW-1485)

### Description
This enables support for the `hearings_substitution_death_dismissal` feature toggle, by which we will control display the "Add Substitution" button for hearing-docket appeals after death dismissal.

### Acceptance Criteria
- [ ] "Add Substitution" button shows/works for `hearing` appeals if feature toggle is enabled
- [ ] Button does not show for `hearing` appeals if feature toggle off

### Testing Plan
1. Confirm that button does not show when feature toggle is not enabled
2. Enable `hearings_substitution_death_dismissal` feature toggle
3. Confirm button shows/works for hearing-docket appeal
